core/src/main/java/org/frankframework/http/AbstractHttpSender.java

```diff
@@ -474,6 +476,32 @@ public String getPhysicalDestinationName() {
 		return getUrl();
 	}
 
+	private HttpSession getSharedSession() {
+		try {
+			return StringUtils.isNotBlank(sharedResourceRef) ? getSharedResource(sharedResourceRef) : null;
+		} catch (NullPointerException e) {
+			return null;
+		}
+	}
+
+	@Override
+	public CredentialFactory getCredentials() {
+		HttpSession sharedSession = getSharedSession();
+		return sharedSession != null ? sharedSession.getCredentials() : super.getCredentials();
+	}
+
+	@Override
+	public String getTokenEndpoint() {
```

Comment:
When you attempt to set this on a sharedResource, throw a ConfiguraitonWarning in configure.